### DISCUSSION TITLE
Backrest for FreeBSD (14)

### DISCUSSION THREAD

**Discussion by Marietto2008** [garethgeorge/backrest#157] [Ideas]
Hello.

I'm a FreeBSD user.  I see that Backrest is not compatible with FreeBSD. Do you plan to make it compatible ?

**Comment by garethgeorge** [comment]
Added a FreeBSD binary, I don't have a system to test on so go ahead and give that a go and let me know if it works for you! https://github.com/garethgeorge/backrest/releases/tag/v0.15.0

**Comment by Marietto2008** [comment]
Thanks it worked here. I've informed the nice guys of the freebsd forums,so that they can test it and report what they like or not :

https://forums.freebsd.org/threads/we-have-a-freebsd-compatible-version-of-backrest.92775/

again,very thanks.
